.github/workflows/release.yaml

```diff
@@ -0,0 +1,338 @@
+name: release
+
+# TODO - the real trigger
+# on:
+#   push:
+#     tags:
+#       - 'v*'
+
+# FAKE for testing
+on:
+  pull_request:
+    paths:
+      - '**/*'
+      - '!docs/**'
+      - '!examples/**'
+      - '!README.md'
+
+
+# TODO - wire up signing for official releasese
+
+jobs:
+  # setup the version for easy consumption in the other jobs
+  version:
+    runs-on: ubuntu-latest
+    steps:
+    # - uses: 'google-github-actions/auth@v2'
+    #   with:
+    #     project_id: 'ollama'
+    #     workload_identity_provider: '${{ secret goes here }}'
+    # - run: |
+    #     gcloud projects list

```

Comment:
```suggestion
```

Reply:
Once we get the secret sorted I'll uncomment this chunk as we'll need it to get signing wired up.